### PR TITLE
Api Wrapper syntax refactor

### PR DESCRIPTION
**Pull request by juanszymczak**

### CODE REVIEW

**Comment by juanszymczak** [comment]
required =D

**Comment by juanszymczak** [comment]
This branch this ready to merge, the errors are rubocop, which will be arranged as they go confirming the PRs.

**Comment by juanszymczak** [comment]
This branch this ready to merge, the errors are rubocop, which will be arranged as they go confirming the PRs.

File: lib/bitex_bot/models/api_wrappers/api_wrapper.rb [hunk 1]
```diff
@@ -15,29 +19,41 @@ class ApiWrapper
     :timestamp, # Integer
     :order # Actual order object
   ) do
-    def method_missing(method_name, *args)
-      order.send(method_name, *args)
+    def method_missing(method_name, *args, &_block)
+      order.send(method_name, *args) || super
```
Comment (tmsrjs) [inline lib/bitex_bot/models/api_wrappers/api_wrapper.rb]: We should forward the block as well: `order.send(method_name, *args, &_block)`

Comment (juanszymczak) [reply lib/bitex_bot/models/api_wrappers/api_wrapper.rb]: done

**Pull request merged**